zipkin-storage/mysql/README.md

```diff
@@ -41,6 +41,38 @@ For example, all the below query the same trace using different tools:
 * zipkin-api: `http://1.2.3.4:9411/api/v1/trace/27960dafb1ea7454?raw`
 * mysql: `select * from zipkin_spans where trace_id = x'27960dafb1ea7454';`
 
+If you are trying to debug from data in the database, it is helpful to
+format IDs as hex, and timestamps as dates. The following is an example
+query which will return one line for each update to a span in the last
+5 minutes.
+
+```sql
+SELECT lower(concat(CASE trace_id_high
+                        WHEN '0' THEN ''
+                        ELSE hex(trace_id_high)
+                    END,hex(trace_id))) AS trace_id,
+       lower(hex(parent_id)) as parent_id,
+       lower(hex(id)) as span_id,
+       name,
+       from_unixtime(start_ts/1000000) as timestamp
```

Comment:
by reformatting into a date, it will be obvious when someone accidentally stores span.timestamp as seconds or millis (instead of micros). It will be obvious as the date will come out as 1970 (and in this query not return at all)